cloud_formation/configs/activities.py

```diff
@@ -12,52 +12,86 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+"""
+Create the activites configuration that consists of
+  * Ingest Lambda
+  * Step Function Activities server ASG
+
+The activities configuration creates all of the Step Function related resources.
+The Activities server run Step Function Activites using the Heaviside library.
+
+As a post-init action the config manually creates the Step Functions by
+compiling the Heaviside files.
+"""
+
+DEPENDENCIES = ['core', 'api', 'redis', 'cachedb']
+
 import time
 import json
-from lib.cloudformation import CloudFormationConfiguration, Ref, Arn, get_scenario, Arg
+from lib.cloudformation import CloudFormationConfiguration, Ref, Arn, Arg
 from lib.userdata import UserData
-from lib.names import AWSNames
+from lib import console
+from lib import utils
 from lib import aws
 from lib import constants as const
 from lib import stepfunctions as sfn
-from update_lambda_fcn import load_lambdas_on_s3, update_lambda_code
-
-keypair = None
-
-
-def create_config(session, domain):
+from lib.lambdas import load_lambdas_on_s3, update_lambda_code
+
+def STEP_FUNCTIONS(bosslet_config):
+    names = bosslet_config.names
+    return [
+        (names.query_deletes.sfn, 'query_for_deletes.hsd'),
+        (names.delete_cuboid.sfn, 'delete_cuboid.hsd'),
+        (names.delete_experiment.sfn, 'delete_experiment.hsd'),
+        (names.delete_coord_frame.sfn, 'delete_coordinate_frame.hsd'),
+        (names.delete_collection.sfn, 'delete_collection.hsd'),
+        #(names.populate_upload_queue.sfn, 'populate_upload_queue.hsd'),
```

Comment:
Do we still need these? Why keep them around?

Reply:
As an example / reminder of a different approach, instead of hard coding the list of AMIs

Reply:
Probably not, I just reorganized what was there.